zutil_p.h

```diff
@@ -11,6 +11,7 @@
 
 #if defined(__APPLE__) || defined(HAVE_POSIX_MEMALIGN)
 #  include <stdlib.h>
+#  include <malloc.h>
```

Comment:
I guess this was not needed, as it does not help. [Manual](https://man7.org/linux/man-pages/man3/posix_memalign.3.html) says we need `#include <stdlib.h>`, which is already included.